.env

```diff
@@ -0,0 +1,36 @@
+
+# Taiga's URLs - Variables to define where Taiga should be served
+TAIGA_SCHEME=http  # serve Taiga using "http" or "https" (secured) connection
+TAIGA_DOMAIN=localhost:9000  # Taiga's base URL
+SUBPATH="" # it'll be appended to the TAIGA_DOMAIN (use either "" or a "/subpath")
+WEBSOCKETS_SCHEME=ws  # events connection protocol (use either "ws" or "wss")
+
+# Taiga's Secret Key - Variable to provide cryptographic signing
+SECRET_KEY="taiga-secret-key"  # Please, change it to an unpredictable value!!
+
+# Taiga's Database settings - Variables to create the Taiga database and connect to it
+POSTGRES_DB=taiga  # PostgreSQL database name
+POSTGRES_USER=taiga  # user to connect to PostgreSQL
+POSTGRES_PASSWORD=taiga  # database user's password
+POSTGRES_HOST=taiga-db  # Docker service hosting PostgreSQL
+#POSTGRES_PORT=5432  # default database port
+
+# Taiga's SMTP settings - Variables to send Taiga's emails to the users
+DEFAULT_FROM_EMAIL=changeme@example.com  # default email address for the automated emails
+EMAIL_BACKEND=console  # use an SMTP server or display the emails in the console (either "smtp" or "console")
+# EMAIL_USE_TLS/EMAIL_USE_SSL are mutually exclusive (only set one of those to True)
```

Comment:
if they are mutually exclusive we should set them as True/False or False/True
Not sure if the comments for SSL and TSL are ok (at least I don't understand them)

Reply:
These comments were extracted from Django. I'd also like to find better comments for those two variables. https://docs.djangoproject.com/en/dev/ref/settings/#std-setting-EMAIL_USE_TLS

Perhaps @bameda can help us here

You're right about the default values, I'll change them to True/False

Reply:
oks, thanks for the clarification!